blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -82,6 +82,10 @@ The following settings are removed:
 Breaking Changes
 ----------------
 
+- Only allow a single port value on each ``discovery.seed_hosts`` (old name
```

Comment:
I don't think this should be a breaking change as the ``discovery.seed_hosts`` setting was not released. I'd rather have this limitation documented

Reply:
`discovery.seed_hosts` is the new name of `discovery.zen.ping.unicast.host` as described in this entry. If a user used a port range for that setting and renamed it according to the upgrade notes, node startup will fail. So it's indeed breaking.